magento/pwa-studio
# [docs] Add site switcher to the header

## Description

This pull request (PR) adds the global navigation menu that links to other documentation sites.

## Related Issue





Closes #ISSUE_NUMBER.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to the documentation page.
2. Verify the menu icon in top right corner shows up.
3. Click the menu icon
4. See the drop-down list of sites appear

## Screenshots / Screen Captures (if appropriate)

![New menu](https://user-images.githubusercontent.com/752610/68770723-4ec25200-05ec-11ea-902f-e0ded2716b5b.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
